server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/crypto/SecurityKeyLoader.java

```diff
@@ -55,9 +55,11 @@ public AsymmetricKeys load() throws Exception {
         Preconditions.checkState(jmapDraftConfiguration.isEnabled(), "JMAP is not enabled");
 
         KeyStore keystore = KeyStore.getInstance(JKS);
-        InputStream fis = fileSystem.getResource(jmapDraftConfiguration.getKeystore());
-        char[] secret = jmapDraftConfiguration.getSecret().toCharArray();
-        keystore.load(fis, secret);
+        char[] secret;
+        try (InputStream fis = fileSystem.getResource(jmapDraftConfiguration.getKeystore())) {
+            secret = jmapDraftConfiguration.getSecret().toCharArray();
```

Comment:
This should be handled in another PR (clearly) but we don't zero the secret char array after using it?

Reply:
Answer, the password is in a String just above, and we are forced by configuration2 to access the secret as a String...